scripts/verify_mdx.js

```diff
@@ -21,12 +28,53 @@ const progressBar = new cliProgress.SingleBar(
 
 const mdxErrors = [];
 
+const createAST = (filePath) => {
+  const mdxFile = fs.readFileSync(filePath);
+
+  const mdxAst = unified()
+    .use(remarkParse)
+    .use(remarkStringify, {
+      bullet: '*',
+      fences: true,
+      listItemIndent: '1',
+    })
+    .use(remarkMdx)
+    .use(remarkMdxjs)
+    .use(remarkFrontmatter, ['yaml'])
+    .parse(mdxFile);
+
+  return mdxAst;
+};
+
+const verifyStepsChildren = (mdxAST) => {
+  let hasNonStepChild = false;
+  let nodeInfo;
+  visit(mdxAST, (node) => {
+    if (node.name !== 'Steps') {
+      return;
+    }
+    hasNonStepChild = node?.children?.some((el) => el.name !== 'Step');
+    nodeInfo = node.position.start;
+  });
+  return { hasNonStepChild, nodeInfo };
+};
+
 const readFile = async (filePath) => {
   let failed = false;
 
   const mdxText = fs.readFileSync(filePath, 'utf8');
   try {
     const jsx = mdx.sync(mdxText);
+    const mdxAst = createAST(filePath);
```

Comment:
i didn't notice this when we looked at it before, but i think we should pass `mdxText` to `createAST` instead of `filePath` so we don't read the file from disk twice